pkg/cli/cc/cc.go

```diff
@@ -142,36 +134,36 @@ func (c *CommonCtx) GetRunningContainers() ([]*types.Container, error) {
 	return topazContainers, nil
 }
 
-func (c *CommonCtx) IsServing(cfg *client.Config) (bool, error) {
-	if c.Config.Defaults.NoCheck {
-		return true, nil
-	}
+// func (c *CommonCtx) IsServing(cfg *client.Config) (bool, error) {
```

Comment:
Overall looks good. Can we remove the `IsServing` func entirely ?

Reply:
done